main.js

```diff
@@ -27,6 +28,9 @@ app.use(sassMiddleware({
   outputStyle: 'compressed'
 }));
 
+// client scripts
+app.use('/js', browserify('./client'));
```

Comment:
Use `path.join` as everywhere else, please. Also, I'd name the directory `client-js`. Or shall we put it in `views/scripts`?